src/components/SelectTime/SelectRegion.jsx

```diff
@@ -0,0 +1,68 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+
+const SelectRegion = () => {
+  const [checkedRegions, setCheckedRegions] = useState([]);
+
+  const handleOnClick = region => {
+    setCheckedRegions([region]);
+  };
+
+  const RegionList = ['홍대', '청담씨네시티', '목동', '피카디리1958', '신촌아트레온'];
```

Comment:
p1) 상수 리스트는 대문자!사용하는게 어떨까요?!
또 저는 상수는 최상단에 빼놓는게 나중에 보기 편하더라구요! 요건 개인 취향이니 참고참고
```suggestion
  const REGION_LIST = ['홍대', '청담씨네시티', '목동', '피카디리1958', '신촌아트레온'];
```